docs/website-docs/device-set-up.md

```diff
@@ -0,0 +1,91 @@
+To record a trace, first let’s get you set up.
+
+These preparatory steps only need to be once.
+
+Android
```

Comment:
Should we separate Android and iOS setup instructions into different files? Especially since there is no iOS self-service